src/main/java/com/minecolonies/coremod/colony/requestsystem/requests/StandardRequestFactories.java

```diff
@@ -34,7 +34,13 @@
     private static final String NBT_CHILDREN  = "Children";
     ////// --------------------------- NBTConstants --------------------------- \\\\\\
 
-    @SuppressWarnings(Suppression.BIG_CLASS)
+    /**
+     * Private constructor to hide the implicit public one.
+     */
+    private StandardRequestFactories()
+    {
+    }
+
     public static final class ItemStackRequestFactory implements IRequestFactory<Stack, StandardRequests.ItemStackRequest>
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce this class from 41 to the maximum allowed 30 or externalize it in a public class. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS2972)